new-feature.txt

```diff
@@ -0,0 +1 @@
+This file was created in the feature1 branch
```

Comment:
It's really good you decided to add a new file in this branch

Reply:
Thanks for the comment